### DISCUSSION TITLE
Welcome to mnlist_website Discussions!

### DISCUSSION THREAD

**Discussion by TwixDUOs** [TwixDUOs/mnlist_website#1] [Announcements]
Hello. Please tell me what I did wrong. I want to learn. Thanks

**Comment by PalmaLuv** [comment]
Hello. 
I suggest to change the cursor in the class [`.nav_link`](https://github.com/TwixDUOs/mnlist_website/blob/6ef6d8d67b20bcdca916d952a748a0e407d339af/assets/css/style.css#L35)
Namely, add the line
```css
cursor: pointer;
```
### What will change
Changes will be visible as cursor change with [`text`](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#:~:text=in%20a%20bitmap.-,text,-The%20text%20can) on [`pointer`](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#:~:text=information%20is%20available.-,pointer,-The%20cursor%20is).
